uach-retrofill.js

```diff
@@ -160,6 +160,14 @@ async function getUserAgentUsingClientHints(hints) {
   });
 }
 
+function getVersion(fullVersionList, majorVersion) {
+  if (!fullVersionList) {
+    return `${majorVersion}.0.0.0`;
+  }
+
+  return fullVersionList.find((item) => item.brand == "Google Chrome")?.version;
```

Comment:
If the `find` fails, do you want to return the same thing as the `!fullVersionList` case above?

Reply:
Oh, that's a great point